test/utils.js

```diff
@@ -104,7 +105,13 @@ const fixtures = {
 		await db.query(
 			`UPDATE user_invites 
 			SET status = $1 WHERE email = $2`, [status, email] 
-		)}
+		)
+	},
+	async generateAccessToken ({data}){
```

Comment:
we won't need the async keyword here and also, let it take `data` parameter without it being in an object